components/styled/icon/src/molecules/icon.tsx

```diff
@@ -83,11 +83,11 @@ const Icon: FC<IconProps> = ({ type, size, status, direction, counterValue, anim
     }
 
     case 'cross': {
-      return <CrossIcon size={size} {...rest} />
+      return <CrossIcon size={size} status={status ?? 'default'} {...rest} />
```

Comment:
You could default this in the `CrossIcon` component to avoid having this wherever we don't want to set status